From 703575f7bbb2dc5356ea77faba5adc237724e543 Mon Sep 17 00:00:00 2001
From: Junde Yhi <lmy441900@aosc.xyz>
Date: Wed, 13 Mar 2019 16:48:10 +0800
Subject: [PATCH] makefile.in: add DESTDIR support

The original makefile is not friendly for distro vendors to
package. Adding DESTDIR support to allow a 2-stage installtion.

Signed-off-by: Junde Yhi <lmy441900@aosc.xyz>
---
 makefile.in | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/makefile.in b/makefile.in
index b2df025..1920f43 100644
--- a/makefile.in
+++ b/makefile.in
@@ -41,14 +41,14 @@ test: vtest27 vtest29 vtest39 vtest615 rstest dtest sumsq_test peaktest
 	./vtest615
 
 install: all
-	mkdir -p @libdir@ 
-	install -m 644 -p $(SHARED_LIB) libfec.a @libdir@
+	mkdir -p $(DESTDIR)@libdir@
+	install -m 644 -p $(SHARED_LIB) libfec.a $(DESTDIR)@libdir@
 #	(cd @libdir@;ln -f -s $(SHARED_LIB) libfec.so)
 	@REBIND@
-	mkdir -p @includedir@
-	install -m 644 -p fec.h @includedir@
-	mkdir -m 0755 -p @mandir@/man3
-	install -m 644 -p simd-viterbi.3 rs.3 dsp.3 @mandir@/man3
+	mkdir -p $(DESTDIR)@includedir@
+	install -m 644 -p $(srcdir)/fec.h $(DESTDIR)@includedir@
+	mkdir -m 0755 -p $(DESTDIR)@mandir@/man3
+	install -m 644 -p $(srcdir)/{simd-viterbi.3,rs.3,dsp.3} $(DESTDIR)@mandir@/man3
 
 peaktest: peaktest.o libfec.a
 	gcc -g -o $@ $^
-- 
2.18.1

